src/signTxPoolRegistration.c

```diff
@@ -806,8 +791,8 @@ static void signTxPoolRegistration_handleOwnerAPDU(uint8_t* wireDataBuffer, size
 			PRINTF("\n");
 
 			subctx->numOwnersGivenByPath++;
-			VALIDATE(subctx->numOwnersGivenByPath <= 1, ERR_INVALID_DATA);
-
+			ctx->poolOwnerByPath = true;
+			memcpy(&ctx->poolOwnerPath, &owner->path, SIZEOF(owner->path));
```

Comment:
```suggestion
			memmove(&ctx->poolOwnerPath, &owner->path, SIZEOF(owner->path));
```

Reply:
This storing of the owner path in ctx should not be done here, only after the security policy has been checked. And we should add an assert before storing that checks that nothing was stored before.

Reply:
We write a lot of stuff into the context, why this is the one you think should happen after policy?

Reply:
You are right, we do VALIDATE after storing stuff in ctx, too. No need to change it then.

But the ASSERT would be reasonable (since we never want to replace an already stored value), woudn't it?

Reply:
The Assert is already there, not sure what version you are seeing.